FirebaseDynamicLinks/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v8.7.1
```

Comment:
Should be 8.8.0 or 8.9.0. See the Milestones tab for upcoming releases

Reply:
Done